src/twisted/web/static.py

```diff
@@ -271,9 +271,31 @@ def ignoreExt(self, ext):
     childNotFound = resource.NoResource("File not found.")
     forbidden = resource.ForbiddenResource()
 
+
     def directoryListing(self):
-        return DirectoryLister(self.path,
-                               self.listNames(),
+        """
+        Return a resource that generates an HTML listing of the
+        directory this path represents.
+
+        @return: A L{resource.Resource} that renders the directory to
```

Comment:
Given that `DirectoryLister` is public, this could refer directly to that.

Reply:
Why is it public though. :sob: